cli/js/lib.deno.ns.d.ts

```diff
@@ -1402,6 +1427,9 @@ declare namespace Deno {
   /** A FileInfo describes a file and is returned by `stat`, `lstat`,
    * `statSync`, `lstatSync`. */
   export interface FileInfo {
+    // TODO(ry) isFile, isDirectory, isSymlink should be getter methods to match
+    // node's api https://nodejs.org/api/fs.html#fs_class_fs_stats
+
```

Comment:
Is there a reason for these to be getter methods when nothing else on this interface is? I get that they need to be kept coherent with each other, but they're still simple constants like everything else. I think we're better off with
```ts
  type: "file" | "directory" | "symlink";
```